File: 1/cs/stringManipulator.cs
using System;

namespace Project
{
    public class StringManipulator
    {
        private readonly string value;

        public StringManipulator(string value)
        {
            this.value = value;
        }

        public static int GetLength(string str)
        {
            return str.Length;
        }

        public static void Main(string[] args)
        {
            Console.Write("Enter a string: ");
            string input = Console.ReadLine();

            int length = GetLength(input);
            Console.WriteLine($"Length of the string: {length}");
        }
    }
}

File: 1/rs/src/string_manipulator.rs
pub struct StringManipulator {
    value: String,
    length: u8,
}

impl StringManipulator {
    pub fn new(value: String) -> StringManipulator {
        let length_value = value.len() as u8;
        StringManipulator{value, length: length_value}
    }

    pub fn get_value(&self) -> (&String, &u8) {
        (&self.value, &self.length)
    }

    pub fn get_length(&self) -> &u8 {
        &self.length
    }

    pub fn get_length_string(value: &str) -> u8 {
        value.len() as u8
    }
}